src/mapml-viewer.js

```diff
@@ -103,7 +103,7 @@ export class MmMapp extends HTMLElement {
     // Set default styles for the map element.
     let mapDefaultCSS = document.createElement('style');
     mapDefaultCSS.innerHTML =
-    `:host {` +
```

Comment:
Particular reason for this?

Reply:
Previously in mm-mapp the mapDefaultCSS was added to the shadowRoot, but for web-map mapDefaultCSS was added to the document head.

I thought I'd keep it consistent with web-map and add mapDefaultCSS to document head in mapml-viewer (previously mm-mapp), with this change :host had to be changed to mapml-viewer.

Any concerns about the change?

Reply:
The inconsistency stems from web-map having a child div as the shadow root (due to limitations) so I couldn't use `:host` there, therefore I had to take the approach of appending to head (as first child to get lower specificity than the author's styles), but I _think_ `:host` is preferable, which was possible for mm-mapp. I believe `:host` is better performance-wise, and this is "critical" CSS for the map element.

Reply:
IIRC `:host` always has lower specificity (similar to UA styles) than the author's styles, which would guarantee the author never have to fight specificity.

Reply:
Alright good points, just reverted that change